qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
I am pretty sure it is impossible to use the cluster API as it is exposed from the SDK today. There are way too many options to setup. This change overall affects only the quality team within Qubole. :)